lib/tasks/manage.rake

```diff
@@ -16,20 +46,33 @@ namespace :manage do
         timeline: "Subscribed #{subscription.created_at}#{subscription.ended_at.present? ? ", Ended #{subscription.ended_at}" : ''}",
       }
     end
-    columns = results.first.each_with_object({}) do |(col, _), h|
-      heading = col.to_s.humanize
-      h[col] = { label: heading, width: [results.map { |g| g[col].size }.max, heading.size].max }
-    end
+    puts hash_to_table(results)
+  end
 
-    # Example output:
-    # | Status                  | SubscriberList            | Frequency | Timeline                                                       |
-    # | Inactive (unsubscribed) | Test foo (slug: test-foo) | daily     | Subscribed 2020-04-18 15:21:20 +0100, Ended 2020-05-12 11:41:04 +0100 |
-    # | Active                  | Bar bar (slug: bar-bar)   | weekly    | Subscribed 2019-06-10 13:48:43 +0100                           |
-    puts "| #{columns.map { |_, g| g[:label].ljust(g[:width]) }.join(' | ')} |"
-    results.each do |h|
-      str = h.keys.map { |k| h[k].ljust(columns[k][:width]) }.join(" | ")
-      puts "| #{str} |"
+  desc "View most recent email delivery attempts for a subscriber"
+  task :view_emails, %i[email_address limit] => :environment do |_t, args|
+    email_address = args[:email_address]
+    limit = args[:limit] || 10
```

Comment:
I think there might be an issue here.  The `args[:limit]` value will be a string, so this needs converting to an integer (see https://deploy.blue.production.govuk.digital/job/run-rake-task/78388/console).